index.js

```diff
@@ -0,0 +1,157 @@
+    (article.source.name === null) ? sourceVar = "" : sourceVar = article.source.name;
+    (article.urlToImage === null) ? imgVar = "jc-gellidon-714673-unsplash.jpg" : imgVar = article.urlToImage;
+    (article.description === null) ? textVar = article.content : textVar = article.description;
+    return `
+    <h2 class="article_title">${article.title}</h2>
+    <div class="image_container"><img class="article_image" src="${imgVar}"></div>
+    <p class="article_content">${textVar}</p>
+    <ul class="article_author_source_list">
+        <li class="author_item">${authorVar}</li>
+        <li class="source_item">${sourceVar}</li>
+    </ul>
+    <a class="article_link" href="${article.url}">Full article available here</a>
+    `
+}
+
+//Appends each div to the content container.
+
+function createNewsContent(news) {
+    news.articles.forEach(item => {
+        if (item.content !== null && item.source.name !== "Bloomberg") {
+            const contentBody = document.querySelector(".news_content");
+            const articleBox = document.createElement("div");
+            articleBox.className = "article_box";
+            articleBox.innerHTML = articleTemplate(item) 
+            contentBody.append(articleBox);
+        }
+    })
+}
+
+//Fetches items for the sidebar
+
+function generateSideBar(url) {
+    fetch(url)
+        .then(response => {
+            // console.log("promise has been resolved")
+            return response.json();
+        }) 
+        .then(body => {
+            // pullNewsObject(body);
+            createSideBarContent(body);
+        });
+    }
+
+
+
+
+//Adds more items to the sidebar as the page number for main article increases
+function updateSideBarUrl(number) {
+    let sidebarUrl = "https://newsapi.org/v2/everything?q=celebrity&q=hollywood&q=showbix&popculture&language=en&totalResults=10&sortBy=publishedAt&apiKey=cd79c8caed5c47f4ae25ec30f7cca674&page=";
+    sidebarUrl = sidebarUrl + number;
+    generateSideBar(sidebarUrl);
+};
+
+
+//Creates template for sidebar articles
+function sideBarTemplate (article) {
+    let authorVar = "";
+    let sourceVar = "";
+    (article.author === null) ? authorVar = "" : authorVar = article.author;
+    (article.source.name === null) ? sourceVar = "" : sourceVar = article.source.name;
+    return `
+    <h2 class="sidebar_title">${article.title}</h2>
+    <div class="sidebar_image_container"><img class="article_image" src="${article.urlToImage}"></div>
+    <ul class="sidebar_source_list">
+        <li class="sidebar_source_item">${sourceVar}</li>
+    </ul>
+    <a class="sidebar_link" href="${article.url}">Full article available here</a>
+    `
+}
+
+//Appends all new content to sidebar
+
+function createSideBarContent(news) {
+    news.articles.forEach(item => {
+        if (item.content !== null && item.source.name !== "Bloomberg") {
+            const sideBar = document.querySelector(".news_sidebar");
```

Comment:
featureBox and sideBar lookup can be taken outside the forEach. Will save you having to run `querySelector` up on each loop iteration.